Dockerfile

```
@@ -8,25 +8,36 @@ RUN --mount=source=.,target=/mnt,type=bind \
                                                vim python3-all pciutils binutils tcpdump \
                                                rsyslog iptables ssl-cert openssl ca-certificates \
                                                curl nmap hydra hping3 iperf3 d-itg socat \
- && apt install -y /mnt/misc/shellinabox_2.21.1_amd64.deb \
- && echo "root:hackinsdn" | chpasswd \
+ && apt install -y /mnt/misc/shellinabox_2.21.1_amd64.deb
+
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN --mount=source=.,target=/mnt,type=bind \
+    cp -vr /mnt/conf/* / \
+ && curl -L --insecure https://trex-tgn.cisco.com/trex/release/v3.05.tar.gz | tar -xz -C /opt/trex
```

I would suggest one separate step only for installing trex which tends to be less prune to changes.
```suggestion
RUN curl -L --insecure https://trex-tgn.cisco.com/trex/release/v3.05.tar.gz | tar -xz -C /opt/trex
```